### PR TITLE
update installation instructions

### PR DESCRIPTION
**Pull request by katmratliff**

### CODE REVIEW

**Comment by katmratliff** [comment]
@hawes1010 I don't have an arduino so I stopped at step #5.

File: Setup_Readme.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@
 Instructions for Installing and running the ISCO_pod_sampler code for the first time
 **********************
 
-1.Download arduino installer-->https://www.arduino.cc/en/Main/Donate
-2.Download Teensyduino that matches your version of Arduino (most likely 1.8.x where x is a number) 
+1.Download and install Arduino-->https://www.arduino.cc/en/Main/Donate
+2.Download and install Teensyduino that matches your version of Arduino (most likely 1.8.x where x is a number)
```
Comment (katmratliff) [inline Setup_Readme.txt:8]: @hawes1010 should we make these install instructions version-agnostic?

Comment (hawes1010) [reply Setup_Readme.txt:8]: I like the change

**Review by hawes1010 — APPROVED** [review]
This clarifies for new users well

**Pull request merged**